src/createRollupConfig.ts

```diff
@@ -16,6 +16,7 @@ const replacements = [{ original: 'lodash', replacement: 'lodash-es' }];
 const babelOptions = {
   exclude: /node_modules/,
   extensions: [...DEFAULT_EXTENSIONS, 'ts', 'tsx'],
+  presets: ['@babel/preset-env'],
```

Comment:
Should this have been `presets: [require.resolve('@babel/preset-env')],`?

Otherwise I am getting
```
UnhandledPromiseRejectionWarning: Error: Cannot find module '@babel/preset-env' ...
```
in the package that consumes tsdx